lms/templates/instructor/instructor_dashboard_2/e-commerce.html

```diff
@@ -464,6 +468,9 @@ <h3 class="error-msgs" id="error-msg"></h3>
      $('#course_price_link').click(function () {
        reset_input_fields();
      });
+     $('#query_registration_code_link').click(function () {
```

Comment:
I can't use the words TODO in the code, because it'll trigger a pylint violation (seriously). Filed: https://openedx.atlassian.net/browse/SOL-956